pyimzml/ImzMLParser.py

```diff
@@ -293,6 +293,9 @@ def get_spectrum_as_string(self, index):
         intensity_string = self.m.read(lengths[1])
         return mz_string, intensity_string
 
+    def getionimage(self, mz_value, tol=0.1, z=None, reduce_func=sum):
+        getionimage(self, mz_value, tol, z, reduce_func)
+
```

Comment:
> make new Imz class that uses the Parser

Yes, and we actually have such a class in `pyImagingMSpec` project: https://github.com/alexandrovteam/pyImagingMSpec/blob/v0.1.2/pyImagingMSpec/inMemoryIMS.py#L187
Currently it loads the whole file into memory and sorts (m/z, intensity, coordinate) tuples by m/z for fast generation of images.

As far as I know, `getionimage` here was intended more as an example of usage rather than part of the interface.